README.md

```diff
@@ -1,8 +1,4 @@
-<h5 align="center">
-  <img src="./.github/assets/logo.png" alt="AsyncAPI logo" width="200">
-  <br>
-  Website
-</h5>
+[![AsyncAPI Website](/public/img/logos/github-repobanner-website.png)](https://www.asyncapi.com)
```

Comment:
For the alt-text, I would make sure to clarify it's a logo/banner. For example right now the alt-text says that the image you added represents the asyncAPI website, but that is not correct. The image you added is our logo and banner. 

Perhaps:

```suggestion
[![AsyncAPI Banner and Logo](/public/img/logos/github-repobanner-website.png)](https://www.asyncapi.com)
```